code/go/0chain.net/smartcontract/storagesc/free_allocation.go

```diff
@@ -264,22 +264,24 @@ func (ssc *StorageSmartContract) freeAllocationRequest(
 			"marshal request: %v", err)
 	}
 
-	assigner.CurrentRedeemed += txn.Value
-	fTxnVal, err := txn.Value.Float64()
+	totalMint, _ := currency.ParseZCN(conf.FreeAllocationSettings.MintAmount)
```

Comment:
We'd better check the error and return if any in case the conf.FreeAllocationSetting is invalid